variables.tf

```
@@ -88,6 +88,12 @@ variable "repository_read_write_access_arns" {
 # Lifecycle Policy
 ################################################################################
 
+variable "create_lifecycle_policy" {
+  description = "Determines whether a lifecycle policy will be created"
+  type        = bool
+  default     = false
```

To be backwards compatible, can we default to `true`

```suggestion
  default     = true
```